.github/workflows/dependabot.yml

```diff
@@ -0,0 +1,55 @@
+name: "Dependabot Workflow"
+on: [pull_request_target] # Dependabot forces this for secrets access
```

Comment:
ℹ️ Using this per https://github.com/dependabot/dependabot-core/issues/3253#issuecomment-852541544